### PR TITLE
Splunk Redesign Comments

### PR DESCRIPTION
**Pull request by anthonythleung**

### CODE REVIEW

File: app/onepassword_events_api/appserver/static/javascript/components/wizard.js [hunk 1]
```diff
@@ -42,14 +59,14 @@ export default function Wizard(props) {
...
-        stepDetails.warning &&
+        currentStep == 1 &&
           (!result.success || result.error) &&
           e(
             "div",
             { className: "warning block" },
             "Your other Splunk apps or add-ons may be able to access your Events API token. Make sure you trust them before you add your token."
           ),
-        stepDetails.redirect &&
+        currentStep == 1 &&
```
Comment (hilaryhacksel) [inline app/onepassword_events_api/appserver/static/javascript/components/wizard.js:69]: Previously these conditions were different, is this step supposed to be the `currentStep == 1`?

**Review by hilaryhacksel — COMMENTED** [review]
Just one small thing :)

File: app/onepassword_events_api/appserver/static/javascript/components/wizard.js (hunk 1 above)
Comment (anthonythleung) [reply app/onepassword_events_api/appserver/static/javascript/components/wizard.js:69]: I removed `stepDetails.redirect` (and `stepDetails.warning`) as they're only used for the first step. It was leftover from when the wizard have more steps so I removed them and just hardcode it to the first step.

**Review by hilaryhacksel — APPROVED** [review]
I think one day we should go a little further and remove the step array entirely. They made a lot of sense before the designs changed so radically and we had many similar steps, but now it's a bit of an awkward abstraction. 

However, I think it's great that this tech debt has been pushed down into only this component and isn't exposed to the rest of the app, so for that reason I think we can move forward 👍

**Pull request merged**